src/Microsoft.Health.Fhir.Shared.Api/Registration/FhirServerServiceCollectionExtensions.cs

```diff
@@ -60,8 +60,7 @@ public static class FhirServerServiceCollectionExtensions
                 .AddNewtonsoftJson(options =>
                 {
                     options.SerializerSettings.DateParseHandling = Newtonsoft.Json.DateParseHandling.DateTimeOffset;
-                })
-                .AddRazorRuntimeCompilation();
```

Comment:
We need this call in there somewhere: 
![image](https://github.com/microsoft/fhir-server/assets/197221/169d2437-6e31-421d-89d9-e1030a0d8398)